static/js/message_flags.js

```diff
@@ -4,12 +4,12 @@ import * as channel from "./channel";
 import * as message_store from "./message_store";
 import * as starred_messages from "./starred_messages";
 
-function send_flag_update(message, flag, op) {
+function update_flag_for_msgs(msg_ids, flag, op) {
```

Comment:
I think it'd be helpful for this to still contain `send` in its name, as I recall in the past some confusion between this function and others that update the UI (after receiving flag updates from elsewhere).

Reply:
Renamed this to `send_flag_update_for_msgs`.